cloudflare/resource_cloudflare_access_mutual_tls_certificate_test.go

```diff
@@ -0,0 +1,136 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccCloudflareAccessMutualTLSBasic(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_mutual_tls_certificate.%s", rnd)
+	cert := os.Getenv("CLOUDFLARE_CERTIFICATE")
```

Comment:
as we've got a few certificate things in the test suite, would it make sense to be more specific with the naming?
```suggestion
	cert := os.Getenv("CLOUDFLARE_MUTUAL_TLS_CERTIFICATE")
```